IFComp/lib/IFComp/Controller/Admin/Voting.pm

```diff
@@ -0,0 +1,158 @@
+package IFComp::Controller::Admin::Voting;
+use Moose;
+use namespace::autoclean;
+use Data::Dumper;
+use JSON;
+
+BEGIN { extends 'Catalyst::Controller'; }
+
+=head1 NAME
+
+IFComp::Controller::Admin::Voting
+
+=head1 DESCRIPTION
+
+A controller for voting reports.
+
+=head1 METHODS
+
+=cut
+
+
+=head2 index
+
+=cut
+
+sub index :Path :Args(0) {
+    my ( $self, $c ) = @_;
+
+    my $comp  = $c->model('IFCompDB::Comp');
+    my $entry = $c->model('IFCompDB::Entry');
+    my $vote  = $c->model('IFCompDB::Vote');
+    my $user  = $c->model('IFCompDB::User');
+
+    my $current_comp;
+    if (my $id = $c->req->param("comp_id")) {
+        $current_comp = $comp->find($id);
+    } else {
+        $current_comp = $comp->current_comp;
+    }
+
+    $c->stash->{ current_comp } = $current_comp;
+    my @available_comps = $comp->search({
+                                         year => {'>=' => 2014},
+                                        },
+                                        { order_by => 'year'}
+                                       )->all;
+
+    $c->stash->{ available_comps } = \@available_comps;
+
+    # Get all entries for this comp
+    my @all_entries = $entry->search({comp => $current_comp->id,
```

Comment:
Instead of checking is_disqualified, please just grab all entries and then grep them through `$_->is_qualified`, which checks other qualification facets as well. (e.g. `my @qualified_entries = grep { $_->is_qualified } @all_entries`